index.js

```diff
@@ -380,6 +380,10 @@ function setResponse(respContext, processed) {
 }
 
 async function InigoFetchGatewayInfo(token) {
+  if (process.env.INIGO_ENABLE === "false") {
```

Comment:
I would recommend more precise name e.g `INIGO_GATEWAY_ENABLE`

Reply:
This is just additional handling for existing env var, it does not introduce new env var. This is the same env var that is used for root InigoPlugin. it's also not possible to have the root plugin disabled and the gateway enabled for example.